webpack.config.js

```diff
@@ -26,6 +26,9 @@ const OUT_PATH = path.resolve('./build');
 const PUBLIC_PATH = '/assets/';
 const IS_DEV = process.env.MDC_ENV === 'development';
 const IS_PROD = process.env.MDC_ENV === 'production';
+const IS_FAST_BUILD = process.env.MDC_BUILD === 'fast';
+const GENERATE_SOURCE_MAPS = IS_DEV && !IS_FAST_BUILD && process.env.MDC_GENERATE_SOURCE_MAPS !== 'false';
```

Comment:
Good point. I removed `MDC_BUILD=fast` and `MDC_DEVTOOL`, so now there's only `GENERATE_SOURCE_MAPS=false`, which seems simpler and more straightforward.

AFAIK, the only env var we currently have is `MDC_ENV=[development|production]`. It's not explicitly documented anywhere, and I'm not sure it needs to be, because I can't think of a reason that a developer would want to change its value.

I'm still experimenting with making our build faster, so I'd prefer not to document `MDC_GENERATE_SOURCE_MAPS` until I'm finished making changes and the API is stable.